core/src/marker/markerManager.cpp

```diff
@@ -63,6 +64,19 @@ bool MarkerManager::setStyling(MarkerID markerID, const char* styling) {
     return true;
 }
 
+bool MarkerManager::setBitmap(MarkerID markerID, int width, int height, const unsigned int* bitmapData) {
```

Comment:
Can we provide another interface which takes in rawData and use stb_image library internally to convert rawData to bitmap?